### DISCUSSION TITLE
Problem with execution

### DISCUSSION THREAD

**Discussion by calrare704** [p-org/P#570] [Q&A]
Hi everyone,

I am working in a project where we are using P. We have been performing some tests with the version of P that is installed with the command:

dotnet tool install --global P

and everything is ok. Additionally, I have tried with the source code and I can compile it successfully. Then, I have compiled the first example (client server example)  but when I tried to check this example I got the following message:

![image](https://user-images.githubusercontent.com/19612649/226973945-97e396c5-b282-4d53-a0e4-c42a89296b72.png)

I do not understand which is the problem. The particular point is I am building P without performing any changes and the installed version of P works perfect.

Many thanks in advance.

**Comment by ankushdesai** [comment]
Thanks for using P. I will get back to you on this in a couple of days.

**Comment by aman-goel** [comment]
@calrare704 

Can you try updating P to the latest P 2.0.5 release and trying again.
You can update P using `dotnet tool update --global P`

**Reply by calrare704** [reply]
Hi @aman-goel,

Thank you for your response. The problem that I reported is related to the version of P that I compiled in my computer. The version of P installed directly with dotnet works ok. It is expected that the compiled version (without any change) works fine and the compilation of the first example (Cliente-Server) is ok but when I tried to perform 'p check' I obtained the error in the image in my previous post. I tried with the version related to tag 2.0.5 (commit acdb157) but I obtained the same error.  The latest version in which the compiled version works ok is the commit 099a4ed2249a5089c36bd3d3906726338113475a. Meanwhile, I have been working with this version.

**Reply by aman-goel** [reply]
Hi @calrare704,

With the latest P 2.0.6 release, this error should be completely eliminated for both dotnet installation and building the latest master from source. Please feel free to reach out if you still see any installation error.
